[2021-06-27 05:55:11,022] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: csv_json_merge.processed_json_check 2021-06-26T00:00:00+00:00 [queued]>
[2021-06-27 05:55:11,029] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: csv_json_merge.processed_json_check 2021-06-26T00:00:00+00:00 [queued]>
[2021-06-27 05:55:11,029] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 05:55:11,029] {taskinstance.py:867} INFO - Starting attempt 1 of 2
[2021-06-27 05:55:11,029] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 05:55:11,039] {taskinstance.py:887} INFO - Executing <Task(BashOperator): processed_json_check> on 2021-06-26T00:00:00+00:00
[2021-06-27 05:55:11,041] {standard_task_runner.py:53} INFO - Started process 4688 to run task
[2021-06-27 05:55:11,095] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: csv_json_merge.processed_json_check 2021-06-26T00:00:00+00:00 [running]> ip-172-31-21-147.ec2.internal
[2021-06-27 05:55:11,108] {bash_operator.py:82} INFO - Tmp dir root location: 
 /tmp
[2021-06-27 05:55:11,110] {bash_operator.py:105} INFO - Temporary script location: /tmp/airflowtmpAYY2ny/processed_json_checkLmLurC
[2021-06-27 05:55:11,110] {bash_operator.py:115} INFO - Running command: shasum ~/airflow/processed/json_data/json*
[2021-06-27 05:55:11,116] {bash_operator.py:122} INFO - Output:
[2021-06-27 05:55:11,137] {bash_operator.py:126} INFO - eb6ab02a5b90aef955b99927403131225abe8f88  /home/ubuntu/airflow/processed/json_data/json_processed_2021-05-16.csv
[2021-06-27 05:55:11,138] {bash_operator.py:130} INFO - Command exited with return code 0
[2021-06-27 05:55:11,145] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=csv_json_merge, task_id=processed_json_check, execution_date=20210626T000000, start_date=20210627T055511, end_date=20210627T055511
[2021-06-27 05:55:21,020] {logging_mixin.py:112} INFO - [2021-06-27 05:55:21,019] {local_task_job.py:103} INFO - Task exited with return code 0
